### PR TITLE
Update metrics to graphiteV2

### PR DESCRIPTION
**Pull request by ddigulescu**
🐿 v2.9.0

### CODE REVIEW

File: lib/metrics.js [hunk 1]
```diff
@@ -90,7 +91,7 @@ Metrics.prototype.init = function (opts) {
 
 	const destination = (noLog) ? {} : {
 		host: FT_GRAPHITE_HOSTNAME,
-		key: FT_GRAPHITE_APIKEY,
+		key: FT_GRAPHITE_UUID,
```
Comment (simonplend) [inline lib/metrics.js]: I think it might be clearer that this is a per-app environment variable if it was named `FT_GRAPHITE_APP_UUID`.

**Comment by simonplend** [comment]
The README needs updating to reflect the introduction of the `FT_GRAPHITE_UUID` environment variable and the removal of `FT_GRAPHITE_APIKEY`.

File: lib/metrics.js (hunk 1 above)
Comment (ddigulescu) [reply lib/metrics.js]: you are right, thanks 👍

**Comment by simonplend** [comment]
Closing as this is branch is very out of date with master now.

**Pull request closed without merging**